src/Ledger/TokenAlgebra.lagda

```diff
@@ -1,40 +1,53 @@
 \section{Token algebras}
+\label{sec:token-algebra}
+
 \begin{code}[hide]
 {-# OPTIONS --safe #-}
-module Ledger.TokenAlgebra where
+
+open import Agda.Primitive  using() renaming (Set to Type)
```

Comment:
This isn't part of the set theory, so you shouldn't rename `Set` to `Type`

Reply:
I thought, for consistency, and to avoid confusion, we agreed to use `Type` in place of `Set` throughout the entire code base.

Reply:
Anyway, I'll change it back to `Set` in `TokenAlgebra` for now.